protobuf/README.md

```diff
@@ -17,28 +17,16 @@ To re-generate the `PulsarApi.java` code you need to apply a patch to the protob
 
 The pre-generated C++ code is at `pulsar-client-cpp/lib/PulsarApi.pb.cc` and `pulsar-client-cpp/lib/PulsarApi.pb.h`.
 
-### Commands for creating the pre-generated files
+You don't need to manually generate C++ code. The C++ code is automatically generated by `cmake`.
+
+### Commands for creating the pre-generated Java code
 
 ```shell
 export PULSAR_HOME=<Path where you cloned the pulsar repo>
 
 cd $HOME
 git clone https://github.com/google/protobuf.git
 
-### For C++ ###
-cd ${HOME}/protobuf
-git checkout v2.6.0
-
-### Compile protobuf
-autoreconf --install
-./configure
-make
-
-### Re-generate PulsarApi
-cd ${PULSAR_HOME}/pulsar-client-cpp/
-export PROTOC=${HOME}/protobuf/src/protoc 
-./generate_protobuf.sh
-
 ### For Java ###
```

Comment:
This line is redundant because the whole section is for Java now.

Reply:
done